packages/docs/src/pages/en/components/alerts.md

```diff
@@ -52,40 +52,95 @@ The recommended placement of elements inside of `v-alert` is:
 | [v-alert](/api/v-alert/) | Primary Component |
 | **v-alert-title** | Functional Component used to display the `v-alert` title. Wraps the `#title` slot |
 
-## Examples
+## Implementation
+
+Unlike the `v-banner` component, `v-alert` is intended to be re-used throughout your application as call-outs that draw a user's attention.
 
 ### Props
 
+In addition to the standard [v-sheet](/components/sheets/) properties such as elevation, dimension, and rounded, `v-alert` supports **v-model**, [variants](), and [density](); and many more.
+
 #### Type
 
-The **type** prop provides 4 default `v-alert` styles: **success**, **info**, **warning**, and **error**. Each of these styles provides a default icon and color. The default colors can be configured globally by customizing [Vuetify's theme](/features/theme).
+The `v-alert` component has 4 contextual states: **success**, **info**, **warning**, and **error**. Each state has a default _color_ and _icon_ associated with it. The following example uses the **type** prop to tell the `v-alert` that it's in an **error** state:
```

Comment:
You use bold/italic text a lot, is there some logic to when you use those vs ``` `` ```?

Reply:
I typically only use ` for components or code related emphasis.